python/llm/example/GPU/Deepspeed-AutoTP-FastAPI/serving.py

```diff
@@ -155,6 +214,35 @@ async def generate(prompt_request: PromptRequest):
             return {"generated_text": output_str}
 
 
+@app.post("/generate_stream/")
+async def generate_stream(prompt_request: PromptRequest):
+    request_id = str(uuid.uuid4())
+    await request_queue.put((request_id, prompt_request))
+    while True:
+        await asyncio.sleep(0.1)
+        if request_id in streamer_dict:
+            token_queue = streamer_dict[request_id]
+            async def output_generator(token_queue):
+                i = 0
+                while i < prompt_request.n_predict:
+                    if not token_queue.empty():
+                        token = await token_queue.get()
+                        logger.error(f"Consumer: {token}")
+                        # yield token
+                        yield json.dumps(
+                            {"request_id": request_id, "text": token}
+                        ) + "\n"
+                    else:
+                        await asyncio.sleep(0.001)
```

Comment:
Is `0.001` a magic setting? Can we align `asyncio.sleep()` to  `asyncio.sleep(0)` or `asyncio.sleep(0.1)`?